pmoves/tests/smoke/test_supabase_selfhosted.py

```
@@ -179,11 +244,12 @@ async def test_supabase_health_check() -> None:
 
         if result.returncode != 0 or "no such" in result.stderr.lower():
             result = subprocess.run(
-                ["docker", "ps", "--filter", f"name={service}", "--format", "{{.Status}}"],
+                ["docker", "ps", "--filter", f"name={service}", "--format", "{{{{.Status}}}}"],
```

Ruff (0.15.5)

[error] Starting a process with a partial executable path

(S607)